src/components/FeatureViews/Registration/CourseList.js

```diff
@@ -226,6 +349,42 @@ const CourseList = ({ filteredCourses, updatedParent }) => {
                 </DialogActions>
             </DialogContent>
         </Dialog>
+        <Dialog open={openInterestDialog} onClose={() => setOpenInterestDialog(false)} PaperProps={{ style: { "height": "310px", "width": "410px", "padding": "32px" } }}>
+                <Grid container spacing={3}>
+                    <Grid item>
+                        <Typography variant="h3">Interested?</Typography>
```

Comment:
Use Dialog's `<DialogTitle/>` , `<DialogContent/>`, and `<DialogContentText/>` instead of `<Grid>` to style the spacing. https://material-ui.com/components/dialogs/